cpp/include/Ice/ServantLocator.h

```diff
@@ -18,10 +17,10 @@ namespace Ice
      * @see ObjectAdapter#findServantLocator
      * \headerfile Ice/Ice.h
      */
-    class ICE_CLASS(ICE_API) ServantLocator
+    class ServantLocator
     {
     public:
-        ICE_MEMBER(ICE_API) virtual ~ServantLocator();
+        virtual ~ServantLocator() = default;
```

Comment:
For the vast majority of our abstraction base classes, we never dynamic_cast to them so I believe this setup is fine.

One exception is NativePropertiesAdmin.